### PR TITLE
Run as service in Ubuntu 16.04 via supervisor

### PR DESCRIPTION
**Pull request by mmarif4u**
As title.

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+### Run as a service in Ubuntu 16.04 LTS
+#### Using supervisor
```
Comment (appleboy) [inline content/doc/installation/run-as-service-in-ubuntu.en-us.md]: Please add an empty line between every section. It is hard to read this file.

**Review by appleboy — CHANGES_REQUESTED** [review]

File: content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Install supervisor by running below command in terminal:
+```sudo apt install supervisor```
+Create a log dir for the supervisor logs(assuming gitea is installed in /home/git/gitea/):
+```mkdir /home/git/gitea/log/supervisor```
+Open supervisor config file in vi/vim/nano etc.
+```sudo vim /etc/supervisor/supervisord.conf```
+And append the following code at the end of the file:
```
Comment (appleboy) [inline content/doc/installation/run-as-service-in-ubuntu.en-us.md]: maybe add reference link: https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea

**Comment by mmarif4u** [comment]
Updated with the requested changes.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by appleboy** [comment]
make L-GTM work

**Comment by lafriks** [comment]
I think this file need something like this at the begining of file
```
---
date: "2017-07-21T12:00:00+02:00"
title: "Run as service in Linux"
slug: "linux-service"
weight: 10
toc: true
draft: false
menu:
  sidebar:
    parent: "installation"
    name: "Linux service"
    weight: 20
    identifier: "linux-service"
---
```

**Review by lafriks — CHANGES_REQUESTED** [review]
See my comment about needed changes

**Comment by mmarif4u** [comment]
@lafriks updated with the requested changes.

**Review by lafriks — APPROVED** [review]

File: content/doc/installation/run-as-service-in-ubuntu.en-us.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+menu:
+  sidebar:
+    parent: "installation"
+    name: "Linux service"
+    weight: 20
+    identifier: "linux-service"
+---
+
+### Run as service in Ubuntu 16.04 LTS  
+#### Using supervisor
```
Comment (bkcsoft) [inline content/doc/installation/run-as-service-in-ubuntu.en-us.md]: I think you need a newline before this row

**Review by bkcsoft — COMMENTED** [review]
Otherwise LGTM

File: content/doc/installation/run-as-service-in-ubuntu.en-us.md (hunk 3 above)
Comment (mmarif4u) [reply content/doc/installation/run-as-service-in-ubuntu.en-us.md]: Do you mean adding new line after Run as service line?.

Comment (lafriks) [reply content/doc/installation/run-as-service-in-ubuntu.en-us.md]: @mmarif4u yes

**Comment by mmarif4u** [comment]
Done.

**Pull request merged**